### PR TITLE
Client page2

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Nice !

File: src/components/ClientPage.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import ImgRessources from "../assets/ImgRessources.jpg";
+
+import {
+  FlexBox,
+  FlexBox1,
+  FlexBox2,
+  Name,
+  Function,
```
Comment (LoicBrassart) [inline src/components/ClientPage.jsx]: I'd advise against this component name, but as long as it's working...

File: src/components/ClientPage.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  RessourcesImg,
+  Agir,
+  AgirFlex,
+  TextAgir,
+  AgirImg,
+  Flash,
+  TextFlash,
+} from "../styled-components/ClientPage";
+
+export default function connexion() {
```
Comment (LoicBrassart) [inline src/components/ClientPage.jsx:23]: Rename > "ClientPage"

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One misname but good otherwise

**Review by Thoms-C — DISMISSED** [review]
Seems ok for me

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Thoms-C — APPROVED** [review]
Clear !

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**